LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016   
+Crown Copyright (c) 2016-2017
```

Comment:
https://www.copyrightservice.co.uk/copyright/p03_copyright_notices is a useful guide